layouts/fastly/helix.vcl

```diff
@@ -327,6 +327,20 @@ sub hlx_github_static_root {
   set req.http.X-Trace = req.http.X-Trace + "(" + req.http.X-Github-Static-Root +  ")";
 }
 
+# Gets the version lock
+sub hlx_version_lock {
+  set req.http.X-Trace = req.http.X-Trace + "; hlx_version_lock";
+  if (req.http.X-OW-Version-Lock) {
+    set req.http.X-Trace = req.http.X-Trace + "(header)";
+    return;
+  }
+
+  set req.http.X-OW-Version-Lock = table.lookup(strain_version_lock, req.http.X-Strain);
+  if (req.http.X-OW-Version-Lock == "") {
+    set req.http.X-OW-Version-Lock = table.lookup(strain_version_lock, "default");
+  }
```

Comment:
@trieloff I need to use this variant instead of the table default value fallback, because it never worked for me. i think the fallback only works, if the entry is missing, but in our case, the entry has the empty string value.

also, the `if (!req.http.X-OW-Version-Lock)` variant also didn't work.... not sure why.